plaidml/exec/ffi.cc

```diff
@@ -125,15 +127,19 @@ plaidml_executable* plaidml_compile(  //
     size_t noutputs,                  //
     plaidml_binding** outputs) {
   return ffi_wrap<plaidml_executable*>(err, nullptr, [&] {
-    IVLOG(1, "Compiling with device: " << device << ", target: " << target);
+    IVLOG(1, "Compiling with device: " << device << ", target: " << target << ", floatx width "
+                                       << program->program->dtypes.floatx.getWidth() << ", intx width "
+                                       << program->program->dtypes.intx.getWidth());
     auto args = BindProgramArguments(program, ninputs, inputs, noutputs, outputs);
     auto exec = std::make_unique<plaidml_executable>();
     std::vector<void*> bufptrs(args.size());
     for (unsigned i = 0; i < args.size(); i++) {
       auto view = args[i].buffer->MapCurrent();
       bufptrs[i] = view->data();
     }
-    exec->exec = std::make_unique<Executable>(program->program->entry, target, *program->program->module, bufptrs);
+
+    exec->exec = std::make_unique<Executable>(program->program->entry, target, *program->program->module, bufptrs,
+                                              program->program->dtypes.floatx, program->program->dtypes.intx);
```

Comment:
I don't think the execution API should have anything to do with arbitrary precision constants. I think we should handle this all during program construction, before we go to compile down to a particular target.

Reply:
Agreed; removed these changes